app/models/intake.rb

```diff
@@ -528,6 +528,22 @@ def triaged_from_stimulus?
     triage_source.present? && triage_source.class == StimulusTriage
   end
 
+  def must_have_document_types
```

Comment:
Non-blocking suggestion: Since these two methods seem to be the same except for `might_have_doc_type?` vs `must_have_doc_type?`, is there a way to unify them a little? Honestly asking; happy as-is with the code.

Reply:
hm, that occurred to me but i couldn't think of anything off the top of my head - maybe that refactor can happen as part of [this story](https://www.pivotaltracker.com/story/show/173349198)...